spacy_llm/tasks/builtin_task.py

```diff
@@ -312,6 +312,14 @@ def _extract_labels_from_example(self, example: Example) -> List[str]:
     def labels(self) -> Tuple[str, ...]:
         return tuple(self._label_dict.values())
 
+    def add_label(self, label: str) -> int:
+        if not isinstance(label, str):
+            raise ValueError(Errors.E187)
```

Comment:
Pretty sure I copied this from spaCy, it might be that we used to allow `int`'s and then we'd put them through the vocab ourselves to do int->str conversion. Either way I figured it didn't hurt to have the same check as in `TrainablePipe` 🤷